### PR TITLE
Send an email on activity creation

### PR DESCRIPTION
**Pull request by stefanmajoor**
Send an email to a predetermined email when an activity is created.

### CODE REVIEW

File: module/Activity/src/Activity/Service/Email.php [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+class Email extends AbstractService
+{
+
+    /**
+     * Send registration email.
+     *
+     * @param NewUserModel $newUser
+     * @param MemberModel $member
+     */
```
Comment (jszanto) [inline module/Activity/src/Activity/Service/Email.php]: a registration email?

File: module/Activity/view/email/activity.phtml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+<?php $lang = 'en'; ?>
+<?= $this->translate('Hello,')?><br /><br />
```
Comment (jszanto) [inline module/Activity/view/email/activity.phtml]: Won't this result in emails being sent in different languages depending on what language the creator of the activity is using?

Comment (stefanmajoor) [inline module/Activity/view/email/activity.phtml]: It indeed would. Can I just make the email in English/Dutch? What has preferences?

Comment (jszanto) [inline module/Activity/view/email/activity.phtml]: I think it is fine to just have the email in one language. Which of the two shouldn't really matter, however the person reading it most likely will be dutch.

Comment (kokx) [inline module/Activity/view/email/activity.phtml]: However, everyone that is dutch can read english emails. Not the other way around.

We can also put 'English text below' at the beginning of the email. And put an english text below the dutch text.

File: module/Activity/src/Activity/Service/Activity.php [hunk 3]
```diff
@@ -314,4 +316,14 @@ public function getActivityMapper()
     {
         return $this->sm->get('activity_mapper_activity');
     }
+
+        /**
+     * Get the email service.
```
Comment (kokx) [inline module/Activity/src/Activity/Service/Activity.php]: Seems like some tab/spaces mismatch here.

**Comment by jszanto** [comment]
Merging this now and then I will fix the tab/spaces issue after that.

**Pull request merged**